selfdrive/car/gm/carcontroller.py

```diff
@@ -46,18 +49,26 @@ def update(self, CC, CS):
     if CS.lka_steering_cmd_counter != self.lka_steering_cmd_counter_last:
       self.lka_steering_cmd_counter_last = CS.lka_steering_cmd_counter
     elif (self.frame % self.params.STEER_STEP) == 0:
-      lkas_enabled = CC.latActive and CS.out.vEgo > self.params.MIN_STEER_SPEED
+      # GM EPS faults on any gap in received message counters. To handle transient OP/Panda safety sync issues at the
+      # moment of disengaging, increment the counter based on the last message known to pass Panda safety checks.
+      idx = (CS.lka_steering_cmd_counter + 1) % 4
+      # Start counting number of resubmits - this should only happen when the panda drops an LKA frame
+      # If we reach 3 attempts to send the same frame, it's time to switch to inactive
```

Comment:
Want to make this a separate PR for the EPS fault fix?

Reply:
Yes